src/components/Timer.jsx

```diff
@@ -1,39 +1,72 @@
 import React, { Component } from 'react';
-import formatTime from '../js/helper';
-
+import { formatTime, alertMessage } from '../js/helper';
+import uuid from 'uuid-v4';
+// 25 min/ 10 min  is default setting
 class Timer extends Component {
 	state = {
+		type: '',
```

Comment:
Maybe we should call it and present it to the user as `category` or something like that? I'm not sure if type is clear enough.

Reply:
make sure. 
I will change it to category